File: Scripts/server/url-decoding/javascript/url-decoding.js
var r =1;

function executeTask(i) {
	decodeURIComponent("http%3A%2F%2Ffoo%20bar%2F")
	return i + 1;
}

for ( var j = 0; j < 10000000; ++j) {
	r = executeTask(j +r);
}

File: Scripts/volatile_and_force_compiler_task/url-decoding/go/url-decoding.go
package main

import (
	"log"
	"net/url"	
	"strconv"
)

func main() {

	for i := 0; i < 100000000; i++ {
		str := strconv.Itoa(i)
		var urlString = "http%3A%2F%2Ffoo%20bar%2F" + str

		u, err := url.QueryUnescape(urlString)
		if err != nil {
			log.Println(err)
		
		}
		_ = u
	}
}
